recipes/asciidoc/meta.yaml

```diff
@@ -0,0 +1,52 @@
+
+{% set name = "asciidoc" %}
+{% set version = "8.6.10" %}
+{% set sha256 = "9e52f8578d891beaef25730a92a6e723596ddbd07bfe0d2a56486fcf63a0b983" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  source_files:
+    - "**"
+  commands:
+    - python asciidoc.py doc/asciidoc.txt  # [py2k]
+
+about:
+  home: https://github.com/asciidoc
+  license: GPL-2.0
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
This can be changed to `license_file: COPYING` and the license file can be dropped from the recipe. That will pick up the license from the source.